client/components/suggestions/index.jsx

```diff
@@ -107,7 +107,10 @@ const Suggestions = React.createClass( {
 	},
 
 	onMouseDown( event ) {
-		this.props.suggest( event.target.textContent );
+		event.stopPropagation();
+		event.preventDefault();
+		//Additional empty space at the end adds fluidity to workflow
+		this.props.suggest( event.target.textContent + ' ' );
```

Comment:
Nope no need to explain further. The comment you left in the code triggered my question